src/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -105,6 +105,7 @@ public function exec(string $command, array $input = []): void
         try {
             $this->_exitCode = $runner->run($args, $io);
         } catch (StopException $exception) {
+            /** @var int $this->_exitCode */
             $this->_exitCode = $exception->getCode();
```

Comment:
Should we use a cast instead of a comment?

Reply:
All our exception classes do actually return int so a casting is necessary, hence the ignore. I'll  try and get the issue sorted out with psalm itself to allow us to override the return type for our `Cake\Core\Exception\Exception` class.

Reply:
The pslam issue is fixed https://github.com/vimeo/psalm/issues/2095.

So once a new release is done we can remove all unnecessary annotations / casting  where `getCode()` is used :slightly_smiling_face:.